main/src/ca/uwaterloo/flix/api/lsp/provider/CompletionProvider.scala

```diff
@@ -119,10 +119,12 @@ object CompletionProvider {
       //
       // Declarations.
       //
-      case SyntacticContext.Decl.Trait => KeywordCompleter.getTraitKeywords
-      case SyntacticContext.Decl.Enum => KeywordCompleter.getEnumKeywords
-      case SyntacticContext.Decl.Instance => InstanceCompleter.getCompletions(context)
-      case _: SyntacticContext.Decl => KeywordCompleter.getDeclKeywords ++ SnippetCompleter.getCompletions(context)
+      case SyntacticContext.Decl.Enum      => KeywordCompleter.getEnumKeywords
+      case SyntacticContext.Decl.Instance  => InstanceCompleter.getCompletions(context) ++ KeywordCompleter.getInstanceKeywords
+      case SyntacticContext.Decl.Struct    => KeywordCompleter.getStructKeywords
+      case SyntacticContext.Decl.Trait     => KeywordCompleter.getTraitKeywords
+      case SyntacticContext.Decl.Type      => KeywordCompleter.getTypeKeywords
+      case SyntacticContext.Decl.OtherDecl => KeywordCompleter.getDeclKeywords ++ SnippetCompleter.getCompletions(context)
```

Comment:
I think we discussed renaming OtherDecl to Module since that is what it is (or should be)

Reply:
Oh, right. I forgot. My bad. I'll get on that